SalesforceCommerceCloud/pwa-kit
# [V2]Generate different reports for different bundles

@W-10921909@


# Description

With our current implementation, we have 5 bundles, but they all get generated into a report.html and stat.json. As a result, the old report will be replaced with the new one. 

This PR adjust webpack config so it will generate different webpack analyzer report for each bundles



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- `pwa-kit-build/src/configs/webpack/config.js`

# How to Test-Drive This PR

- run `npm run analyzer-build` at `packages/pwa`
- Verify that there will be 5 tabs opening for each report. 
- Verify it works on common browsers like Chrome, Safari, and Firefox. You need to set each as default browser, so the tab will automatically open in those browsers. 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
